lib/mercator.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * @typedef {Object} Point
+ * @property {Number} x
+ * @property {Number} y
+ */
+
+/**
+ * @typedef {Object} LatLng
+ * @property {Number} lat
+ * @property {Number} lng
+ */
```

Comment:
These two should already be defined as `google.maps.Point` and `google.maps.LatLngLiteral`. If that doesn't give work with Intellisense, you may need to add `/// <reference types="google.maps" />` somewhere. That's already in `MVTSource.js`, but maybe it won't apply in a different file.